docs/disruption_diagram_alternate_structure_examples.md

```diff
@@ -0,0 +1,159 @@
+# Disruption diagram alternate structure examples
+
+The alternate structure factors out some data (e.g. line color) to the top level, and
+leans more on the client to determine the appropriate presentation of symbols, line segments, etc.
+rather than directly telling the client what asset to use for each piece of the diagram.
+
+## Example 1: Shuttle from Boston College to Babcock St, screen at Government Center
+![image](assets/disruption_diagrams/1.png)
+
+```ts
+{
+  effect: "shuttle",
+  line: "green",
+  current_station_slot_index: 11,
+  effect_region_slot_index_range: [0, 8],
+  slots: [
+    {type: "terminal", label_id: "place-lake"},
```

Comment:
What is the advantage of passing the `stop_id` to the client for terminals? Could we just respond with the station's name instead?

Reply:
Yeah, any value that's unique-per-station should work. Stop/station IDs are already unique so I figured we could use those.

I also want to check with design/product whether we could come up with some consistent algorithm for the end label presentation, so that we don't need to hardcode each one. If we can, then using the station's name would let us keep sending the same data from the server and just use it differently on the client.